gateway/webhook/webhook.go

```diff
@@ -122,6 +125,53 @@ func (webhook *HandleT) RequestHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var postFrom url.Values
+	var multipartForm *multipart.Form
+
+	if r.Method == "GET" {
+		return
+	}
+	contentType := r.Header.Get("Content-Type")
+	if contentType == "application/x-www-form-urlencoded" {
```

Comment:
do we need to do a case insensitive comparison or does the http package return it in lowercase irrespective of what the user set